src/test/fuzz/tx_decompression.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2019-2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/util/setup_common.h>
+#include <primitives/transaction.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <rpc/request.h>
+#include <validation.h>
+#include <rpc/client.h>
+#include <rpc/server.h>
+#include <util/rbf.h>
+#include <univalue.h>
+#include <rpc/util.h>
+#include <core_io.h>
+#include <version.h>
+#include <logging.h>
+#include <cassert>
+#include <cmath>
+
+namespace {
+    struct TXDecompressionFuzzTestingSetup : public TestingSetup {
+    	TXDecompressionFuzzTestingSetup(const ChainType& chain_type, const std::vector<const char*>& extra_args) : TestingSetup{chain_type, extra_args}
+    	{}
+
+        UniValue CallRPC(const std::string& rpc_method, const std::vector<std::string>& arguments)
```

Comment:
In c6f4427153b02f4b66535c9b8fdb0e9a2efa35cc:

Not our highest priority, but before we submit this to Core we need to drop all the RPC stuff from the fuzzer and just inline the compression/decompression stuff.

Reply:
Removed rpc tests in favor of compression roundtrip, Thanks!